data/bhattabhi013.json

```
@@ -0,0 +1,62 @@
+{
+    "name": "Abhishek Bhatt",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Full-Stack developer | Flutter | NodeJs | Angular",
+    "avatar": "https://github.com/bhattabhi013.png",
+    "tags": ["flutter", "nodejs", "angular", "javascript", "typescript", "open-source"],
+    "socials": [
+      { "platform": "twitter", "url": "https://twitter.com/AbhishekBtwt" },
+      { "platform": "github", "url": "https://github.com/bhattabhi013" },
+      { "platform": "instagram", "url": "https://www.instagram.com/bwith_abhi" }
+    ],
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/bhattabhi013",
+        "icon": "github"
+      },
+      {
+        "name": "Check out my Portfolio",
+        "url": "https://bhattabhi013.github.io./",
+        "icon": "link"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/AbhishekBtwt",
+        "icon": "twitter"
+      },
+      {
+        "name": "Connect with me on Linkedin",
```

Non-blocking: I recommend capitalizing the `i` here. What do you think?

```suggestion
        "name": "Connect with me on LinkedIn",
```